release.sh

```diff
@@ -32,24 +32,26 @@ fi
 
 # prompt error if non-supported event
 if [ ${GITHUB_EVENT_NAME} == 'release' ]; then
-    echo "Event: ${GITHUB_EVENT_NAME}"
+  echo "Event: ${GITHUB_EVENT_NAME}"
 elif [ ${GITHUB_EVENT_NAME} == 'push' ]; then
-    echo "Event: ${GITHUB_EVENT_NAME}"
+  echo "Event: ${GITHUB_EVENT_NAME}"
 elif [ ${GITHUB_EVENT_NAME} == 'workflow_dispatch' ]; then
-    echo "Event: ${GITHUB_EVENT_NAME}"
+  echo "Event: ${GITHUB_EVENT_NAME}"
 elif [ ${GITHUB_EVENT_NAME} == 'workflow_run' ]; then
-    echo "Event: ${GITHUB_EVENT_NAME}"
+  echo "Event: ${GITHUB_EVENT_NAME}"
+elif [ ${GITHUB_EVENT_NAME} == 'schedule' ] && [ ${INPUT_ALLOW_SCHEDULE^^} == 'TRUE' ]; then
```

Comment:
In my option, add the `allow_schedule` arg is a little bit redundant and complicated, just add one more line here for `schedule` check it should be good enough.